### ISSUE TITLE
Customize day and night theme in settings

### ISSUE THREAD

**Issue by kompot** [gilday/dark-mode-sync-plugin#2]
Hey,
it would be great to be possible to customize theme for both modes via some kind of a drop down.

**Comment by danielgarciagarcia** [comment]
Agreed! I love this plugin but dislike Darcula theme, I prefer One Dark theme.

**Comment by bradhvr** [comment]
I'd love to see this feature added.

**Comment by hy0ug0** [comment]
@gilday Hi there, there is a thread about this feature here https://youtrack.jetbrains.com/issue/IDEA-203554 maybe we could help you to build it ?

**Comment by gilday** [comment]
Hi @Hugo-Contreras, I would like to add this feature, but doing so has been competing with my other obligations. That being said, I started building this out last night. I'll share my wip on a branch.

I believe we just need a simple preferences page with two drop-downs for selecting a light and a dark theme - anything else?

Of course, if someone else gets this working first, contributions are welcome.

**Comment by hy0ug0** [comment]
@gilday Yes that's all we need, to be able to choose which theme to apply for each case. I'll look at your WIP branch when you push it.

**Comment by steebchen** [comment]
It would be awesome if we could even select a specific editor theme depending on the mode in addition to the IDE theme.

**Comment by kannicht** [comment]
Hi everybody!

@gilday I already started working on the matter. I'll open a PR once im done.

@steebchen You may mean the color scheme of the editor?

**Comment by gilday** [comment]
@kannicht is there an opportunity to combine our works in progress? I have the light / dark theme preferences working in [this branch](https://github.com/gilday/dark-mode-sync-plugin/tree/2-customize-dark-and-light-themes). IDEA plugins are not my expertise, so while it seems to work on my machine, I wouldn't be surprised to learn that you've uncovered something in your work-in-progress that I've missed.

In addition to some general clean-up, I was hoping to add some tests to it before shipping it. It lacks the Editor Theme support that @steebchen requested - I wasn't aware Editor Theme was different from Appearance Theme.

**Comment by kannicht** [comment]
HI @gilday !

I had a look at your approach to the matter. It looks more practical than mine. As far as I can tell it works pretty good. 

There is only one issue i ran into while running runIDE:
```

2020-01-02 15:51:52,315 [    925]   WARN - i.mac.MacOSApplicationProvider - com/apple/eawt/AppEvent$AboutEvent 
java.lang.NoClassDefFoundError: com/apple/eawt/AppEvent$AboutEvent
	at com.intellij.ui.mac.MacOSApplicationProvider$Worker.initMacApplication(MacOSApplicationProvider.java:57)
	at com.intellij.ui.mac.MacOSApplicationProvider.initApplication(MacOSApplicationProvider.java:42)
	at com.intellij.idea.ApplicationLoader.startApp(ApplicationLoader.kt:159)
	at com.intellij.idea.ApplicationLoader.executeInitAppInEdt(ApplicationLoader.kt:93)
	at com.intellij.idea.ApplicationLoader.access$executeInitAppInEdt(ApplicationLoader.kt:1)
	at com.intellij.idea.ApplicationLoader$initApplication$1$1.run(ApplicationLoader.kt:355)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.lang.ClassNotFoundException: com.apple.eawt.AppEvent$AboutEvent
Caused by: java.lang.ClassNotFoundException: com.apple.eawt.AppEvent$AboutEvent

	at com.intellij.util.lang.UrlClassLoader.findClass(UrlClassLoader.java:307)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 19 more
```

I don't think it compromises the overall functionality of the plugin.

**Comment by Sottti** [comment]
@gilday @kannicht thanks a lot for working on this! Been literally looking for this the last month! Been able to customize themes is a must for me because I don't use the default themes! So Thanks

**Comment by hy0ug0** [comment]
@gilday Hi there, I looked at your code and it seems to work pretty well, do you want to add more things before merging it ?
I also worked on an other feature to pick a random theme on startup and when the dark mode is enabled, I will create an other branch for that.

**Comment by kannicht** [comment]
I'd suggest to solve issue #7 first before implementing the feature. I'm on that. I'm going to fix it soon.

I don't think a random theme is a productivity win. To me it is just playing around with the feature itself. @Hugo-Contreras

**Comment by kannicht** [comment]
As issue #7 doesn't seem to bee a problem I'd suggest to merge the Preference pane into develop and furthermore to master as a release. @gilday

**Comment by gilday** [comment]
Thanks for testing. I was holding off on merging, because I noticed some weird behavior where the theme would half-change. I don't think this is a regression, and I'm glad that the change has been tested on more than just my machine, so I'll ship it tonight

**Comment by gilday** [comment]
Shipped in 1.1.0
